src/main/java/edu/pav/PatientTrackerSystem/commons/dto/DoctorSignupRequest.java

```diff
@@ -0,0 +1,33 @@
+package edu.pav.PatientTrackerSystem.commons.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@AllArgsConstructor
+@SuperBuilder
+public class DoctorSignupRequest {
+
+    @JsonProperty(value = "user_name")
+    private String userName;
```

Comment:
This might be redundant.  Current request body was 

{
    "dob": "1979-06-03",
    "name": "Dr. Bobert Tea",
    "hospital": "Downside Medical",
    "speciality": "Neurosurgeon",
    "address": "246 Tree St, Countryside",
    "phoneNumber": "444-507-8901",
    "email": "dr.bobert.coffee@example.com",
    "password": "bobertea"
}